web-app/src/app/app.component.ts

```diff
@@ -46,5 +47,10 @@ export class AppComponent implements OnInit {
         this.modalSrv.closeAll();
       }
     });
+    // 主题设置
```

Comment:
All comments should be translated in English
@keaifafafa

Reply:
ok